sw/device/tests/sim_dv/chip_sw_lc_ctrl_scrap_impl.h

```diff
@@ -1,12 +1,12 @@
-// Copyright Nuvoton contributors.
-// Licensed under the Apache License, Version 2.0, see LICENSE for details.
-// SPDX-License-Identifier: Apache-2.0
-
-#ifndef OPENTITAN_SW_DEVICE_TESTS_SIM_DV_LC_CTRL_SCRAP_IMPL_H_
-#define OPENTITAN_SW_DEVICE_TESTS_SIM_DV_LC_CTRL_SCRAP_IMPL_H_
-
-#include <stdbool.h>
-
-bool execute_lc_ctrl_scrap_test(bool use_ext_clk);
-
-#endif  // OPENTITAN_SW_DEVICE_TESTS_SIM_DV_LC_CTRL_SCRAP_IMPL_H_
\ No newline at end of file
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_TESTS_SIM_DV_CHIP_SW_LC_CTRL_SCRAP_IMPL_H_
+#define OPENTITAN_SW_DEVICE_TESTS_SIM_DV_CHIP_SW_LC_CTRL_SCRAP_IMPL_H_
+
+#include <stdbool.h>
+
+bool execute_lc_ctrl_scrap_test(bool use_ext_clk);
```

Comment:
Given that `hip_sw_lc_ctrl_scrap_test.c` and `chip_sw_lc_ctrl_scrap_impl.h` are just boiler plate files without much content, I suggest to just put the test implementation into one file `chip_sw_lc_ctrl_scrap_test.c`.